specification/communication/data-plane/Identity/stable/2022-10-01/CommunicationIdentity.json

```diff
@@ -87,7 +115,7 @@
           }
         },
         "x-ms-examples": {
-          "Delete an identity": {
+          "Delete an identity with optional idempotent processing using Repeatability-Request-ID and Repeatability-First-Sent headers. If the headers are not provided, the request will be processed in a non-idempotent way.": {
```

Comment:
I wonder if we need to say that it will be processed in a non-idempotent way for delete and issue/revoke tokens where the execution is naturally idempotent. But since this is just a comment on the examples and not the route summary itself, I don't have a strong opinion.

Reply:
I put back original comments for "naturally idempotent" methods.